src/constructs/loadbalancing/alb/application-listener.test.ts

```diff
@@ -246,4 +246,34 @@ describe("The GuHttpsApplicationListener class", () => {
       ],
     });
   });
+
+  test("uses HTTPS protocol by default", () => {
+    const stack = simpleGuStackForTesting();
+
+    new GuApplicationListener(stack, "Listener", {
+      loadBalancer: getLoadBalancer(stack),
+      defaultTargetGroups: [getAppTargetGroup(stack)],
+      certificates: [{ certificateArn: "arn:aws:acm:eu-west-1:000000000000:certificate/123abc-0000-0000-0000-123abc" }],
+      ...app,
+    });
+
+    expect(stack).toHaveResource("AWS::ElasticLoadBalancingV2::Listener", {
+      Protocol: "HTTPS",
+    });
+  });
+
+  test("Can override default protocol with prop", () => {
```

Comment:
Minor: I would probably group this with [the other `GuApplicationListener` tests](https://github.com/guardian/cdk/blob/8125805fe5ece34996da2e031213e30a9a43711f/src/constructs/loadbalancing/alb/application-listener.test.ts#L36-L143).

Reply:
Good shout, thanks.